### ISSUE TITLE
MO High Contrast Issues

### ISSUE THREAD

**Issue by becka11y** [readium/readium-js-viewer#170]
1) The MO highlight is not visible in Windows high contrast mode.  This can be fixed by removing:
border: transparent ! important; (or changing it to border: backgroundColor !important;
and adding
border-bottom: 1px dotted backgroundColor !important
for the mo-overlay default style. This does leave a bit of the radius visible but we can live with that rather than having separate classes for high contrast. Or we could remove one of the color options and add a high contrast specific style option in that adds a bottom border and removes the border-radius.

2) The highlight color options in the cloud reader should be actual color names rather than #1,#2, etc.(not critical but useful for people with color issues to help them make the best selection).

3) Consider using real images for the color swatches rather than background color (which disappears in high contrast mode) for the buttons.  Not critical since high contrast mode will rely on the underline added in above in 1) rather than the actual color.

This is a V1+ issue as it only affects IE and FF on Windows.
